applications/osb-portal/webpack.common.js

```diff
@@ -71,6 +71,7 @@ module.exports = env => {
 
         plugins: [
             new CleanWebpackPlugin(),
+            new webpack.EnvironmentPlugin({ 'DOMAIN': env.DOMAIN, 'NAMESPACE': env.NAMESPACE }),
```

Comment:
this only works in case you always build your images in all environments
with deploying images from a registry this breaks

Reply:
I'm not getting this..

Reply:
the env.DOMAIN is set by the codefresh task so when you use pre-build images of osb to deploy in e.g. staging the build is not run so the DOMAIN variable is not used so it will link to the DOMAIN set during the build (test)

Reply:
DOMAIN and NAMESPACE variables are set in the pods

Reply:
this is working already on v2 deployment: https://www.v2.opensourcebrain.org/keycloak.json

Reply:
Ah yes, I now remember.

in the entrypoint.sh we replace the placeholders